tests/test_server.py

```diff
@@ -206,6 +206,7 @@ def test_srv_keep_alive(srv):
     assert not srv._keep_alive
 
 
+@pytest.mark.timeout(0)
```

Comment:
As I see tests are passed both locally and on travis-ci.
If timeout is an issue it should be fixed by separate PR anyway